pkg/app/master/command/images/handler.go

```diff
@@ -126,6 +128,11 @@ func OnCommand(
 	images, err := crtClient.ListImages(cparams.Filter)
 	xc.FailOn(err)
 
+	if cparams.TUI {
```

Comment:
We'll need a bit of extra logic to configure `xc` before `OnCommand` is called, so it knows it's executing in the TUI mode. That way the ```xc.Out.Info("cmd.input.params",
		ovars{
			"runtime": rr,
			"filter":  cparams.Filter,
		})``` call earlier in `OnCommand` won't print stuff to console.

This snippet in `cli.go` for the command will need to be enhanced to know we are in the TUI mode: ```xc := app.NewExecutionContext(
			Name,
			gcvalues.QuietCLIMode,
			gcvalues.OutputFormat)```